examples/kubernetes/travel-agency/data-index.yaml

```diff
@@ -1,12 +1,31 @@
+#Strimzi operator should be pre-installed in namespace
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoInfra
+metadata:
+  name: kogito-kafka-infra
+spec:
+  resource:
+    apiVersion: kafka.strimzi.io/v1beta1
+    kind: Kafka
+---
+#Infinispan operator should be pre-installed in namespace
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoInfra
+metadata:
+  name: kogito-infinispan-infra
+spec:
+  resource:
+    apiVersion: infinispan.org/v1
+    kind: Infinispan
+---
 apiVersion: app.kiegroup.org/v1alpha1
 kind: KogitoDataIndex
 metadata:
   name: data-index
 spec:
   replicas: 1
   image: quay.io/kiegroup/kogito-data-index:0.12
```

Comment:
Please update the image version too

Reply:
Same as https://github.com/kiegroup/kogito-cloud-operator/blob/master/examples/kubernetes/travel-agency/management-console.yaml#L7
Or possibly get rid of the version to use latest available.

Reply:
I had removed tag from image so that it could refer latest